Library/Homebrew/cmd/gist-logs.rb

```diff
@@ -1,3 +1,14 @@
+#:  * `gist-logs` [`--new-issue`|`-n`] <formula>
+#:     Upload logs for a failed build of <formula> to a new gist.
+#:
+#:     <formula> is usually the name of the formula to install, but it can be specified
+#:     several different ways. See [SPECIFYING FORMULAE].
```

Comment:
I think this needs to be `[SPECIFYING FORMULAE][]` to be recognized and rendered correctly, but hard to tell given the many spurious changes in the current version of the man page and HTML version of it.

Also: “specified several different ways” ➡️ “specified **in** several different ways”.